frontend/src/component/user/Profile/PersonalAPITokensTab/CreatePersonalAPIToken/CreatePersonalAPIToken.tsx

```diff
@@ -226,6 +250,7 @@ export const CreatePersonalAPIToken: FC<ICreatePersonalAPITokenProps> = ({
                             type="submit"
                             variant="contained"
                             color="primary"
+                            disabled={!isValid}
```

Comment:
I'm always a little on on the fence about disabled buttons due to accessibility concerns (see [this CSS-tricks article](https://css-tricks.com/making-disabled-buttons-more-inclusive/) for why), so I'm wondering whether we can make this more accessible somehow?

Reply:
This is a pattern that we're using all throughout the frontend. We can definitely make this better but I don't think we should do that in this PR. Very open to suggestions @thomasheartman but it's more work than just this piece of work

Reply:
Yeah, that's fair. I'm aware that we do this other places (got an [old trello card for it](https://trello.com/c/ppggUlV6/297-dont-use-the-disabled-attribute-on-buttons), so I'm on board with that. But I figured that when we introduce new changes, we might want to shift and gradually shift over 💁🏼  Very open to being overruled, though 😄

Reply:
Ye, I don't mind doing a gradual shift over, it's a pretty good idea actually! But before we do, I kinda want a plan to know what we want to achieve and how; and we should do that on features that can sit for a while - PATs are on our critical path for some internal stuff so I'd like to try to keep the work as focused as possible

Reply:
I think this is an interesting discussion and thanks for sharing the article!
If `MUI` default behaviour does not cover this accessibility requirement (apparently not) then I agree that this belongs in a separate PR that pinpoints the exact behaviour that we want, creates a component that implements it, and probably changes every button to use our new component - Or we follow a gradual shift approach after that PR, which I think is more reasonable.